elixir/apps/domain/lib/domain/config/clients_upstream_dns.ex

```diff
@@ -0,0 +1,105 @@
+defmodule Domain.Config.ClientsUpstreamDNS do
```

Comment:
put to `configuration/`
```suggestion
defmodule Domain.Config.Configuration.ClientsUpstreamDNS do
```

Reply:
This has been moved.